packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -62,7 +62,7 @@ const StyledTooltip = styled.span`
     display: block;
     right: 0;
     left: 0;
-    height: 8px;
+    height: var(--overlay-offset, 0.25rem);
```

Comment:
@langermank should this be 4px or 0.25rem?

Reply:
What you have is perfect!